src/screens/TopicPageScreen/elements/ButtonFollowing.js

```diff
@@ -28,14 +28,14 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     paddingVertical: 6,
     borderWidth: 1,
-    borderColor: followType === 'incognito' ? COLORS.anon_secondary : COLORS.signed_primary,
+    borderColor: followType === 'incognito' ? COLORS.anon_primary : COLORS.signed_primary,
     borderRadius: 8
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderColor for the 'incognito' followType has been changed from `COLORS.anon_secondary` to `COLORS.anon_primary`. Ensure that this color change aligns with the overall design and accessibility guidelines, as it could impact visibility and user experience.

```diff
-    borderColor: followType === 'incognito' ? COLORS.anon_secondary : COLORS.signed_primary,
+    borderColor: followType === 'incognito' ? COLORS.anon_primary : COLORS.signed_primary,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->